mitosheet/setup.py

```diff
@@ -121,7 +121,8 @@
                 'pytest',
                 'flake8',
                 'types-chardet',
-                'mypy'
+                'types-requests',
```

Comment:
Do we use this? Do we need this in the deployed package (I think no!) or just the development version?

Reply:
Right now it's just in the `test` section. I think that section is just used in development and not the deployed package, right? That's where we have mypy, pytest, and types-charset, none of which are needed in the deployed package.